### PR TITLE
Switch to tiles

### PR DESCRIPTION
**Pull request by klizhentas**
Switch to tiles from bullet lists.
Focus user attention on products, remove extra text.
List popular use-cases for developers and security teams.

### CODE REVIEW

File: docs/pages/index.mdx [hunk 1]
```diff
@@ -6,22 +6,32 @@ h1: Introduction
...
-- It proxies and inspects SSH, Kubernetes, Web, and Database protocols.
-  For example for SSH, it controls the session from the start
-  and captures a session recording and in-kernel system calls using BPF.
-- It removes a need for VPN and can connect multiple regions and organizations
-  in a decentralized network using mutual TLS and SSH tunnels.
+<TileSet>
+  <Tile icon="bolt" title="Server Access" href="./getting-started.mdx">
+    Single Sign-On, short lived certificates and audit for SSH servers.
+  </Tile>
+  <Tile icon="window" title="Application Access" href="./getting-started.mdx">
```
Comment (r0mant) [inline docs/pages/index.mdx:19]: Update href to point to app access intro.

File: docs/pages/index.mdx [hunk 2]
```diff
@@ -6,22 +6,32 @@ h1: Introduction
...
+    Provide secure access to internal dashboards and web applications.
+  </Tile>
+  <Tile icon="kubernetes" title="Kubernetes Access" href="./kubernetes-access/introduction.mdx">
+    Single Sign-On, audit and unified access for Kubernetes clusters.
+  </Tile>
+  <Tile icon="database" title="Database Access" href="./database-access/introduction.mdx">
+    Secure access to PostgreSQL and MySQL databases.
+  </Tile>
+  <Tile icon="cloud" title="Cloud" href="./cloud/introduction.mdx">
+    Secure access to PostgreSQL and MySQL databases.
```
Comment (r0mant) [inline docs/pages/index.mdx:29]: Update text to be Cloud specific.

**Review by r0mant — APPROVED** [review]

File: docs/pages/installation.mdx [hunk 3]
```diff
@@ -208,3 +208,24 @@ $ export arch=amd64 # '386' 'arm' on linux or 'amd64' for all distros
...
+
+## Operating System Support
+
+Teleport is officially supported on the platforms listed below. It is worth noting
+that the open source community has been successful in building and running Teleport on
+UNIX variants other than Linux \[1].
+
+| Operating System | Teleport Client | Teleport Server |
+| - | - | - |
+| Linux v2.6+ | yes | yes |
```
Comment (russjones) [inline docs/pages/installation.mdx:221]: Go minimum requirements are 2.6.23 and above. For example 2.6.18 (RHEL/CentOS 5) won't work.

https://github.com/golang/go/wiki/MinimumRequirements

**Review by russjones — APPROVED** [review]

File: docs/pages/index.mdx [hunk 4]
```diff
@@ -6,22 +6,32 @@ h1: Introduction
...
-- Teleport replaces legacy keys and shared secrets with short-lived X.509 and SSH certificates
-  for services and users.
-- It proxies and inspects SSH, Kubernetes, Web, and Database protocols.
-  For example for SSH, it controls the session from the start
-  and captures a session recording and in-kernel system calls using BPF.
-- It removes a need for VPN and can connect multiple regions and organizations
-  in a decentralized network using mutual TLS and SSH tunnels.
+<TileSet>
+  <Tile icon="bolt" title="Server Access" href="./getting-started.mdx">
+    Single Sign-On, short lived certificates and audit for SSH servers.
```
Comment (inertial-frame) [inline docs/pages/index.mdx:17]: This is great and a much better improvement overall!

* Nit: `Single Sign-On, short-lived certificates, and audit for SSH servers.`

File: docs/pages/index.mdx [hunk 5]
```diff
@@ -6,22 +6,32 @@ h1: Introduction
...
+  </Tile>
+  <Tile icon="window" title="Application Access" href="./application-access/introduction.mdx">
+    Provide secure access to internal dashboards and web applications.
+  </Tile>
+  <Tile icon="kubernetes" title="Kubernetes Access" href="./kubernetes-access/introduction.mdx">
+    Single Sign-On, audit and unified access for Kubernetes clusters.
```
Comment (inertial-frame) [inline docs/pages/index.mdx:23]: Nit: `Single Sign-On, audit, and unified access for Kubernetes clusters.` 

Oxford comma here.

File: docs/pages/index.mdx [hunk 6]
```diff
@@ -32,58 +42,44 @@ With Teleport you can:
...
+  <TileListItem href="https://goteleport.com/get-started/">
+    Reach out sales for a demo of Teleport Enterprise.
+  </TileListItem>
+  </TileList>
+  <TileList title="For security teams" icon="lock">
+    <TileListItem href="./openssh-teleport.mdx">
+     Capture sessions and manage certificates for existing OpenSSH fleet.
+    </TileListItem>
+    <TileListItem href="./architecture/authentication.mdx#issuing-user-certificates">
+      Replace static keys and passwords with short lived certificates.
```
Comment (inertial-frame) [inline docs/pages/index.mdx:65]: Nit: `short-lived`

**Review by inertial-frame — APPROVED** [review]
Great improvement overall. Helps to bring together the page and highlight our best features!

Minor editorial comments (non-blocking) only. LGTM

**Pull request merged**